server/pbench/lib/config/pbench-server-satellite.cfg.example

```diff
@@ -0,0 +1,29 @@
+# This file describes the elements of deploying the pbench background tools.
+[DEFAULT]
+# cvalues here override those in pbench-server-default.cfg.
```

Comment:
Is this a typo? What does "cvalues" mean?

Reply:
Changed to "The values ...".